appinfok8s.go

```diff
@@ -0,0 +1,60 @@
+package appwrap
+
+import (
+	"context"
+	"fmt"
+	"os"
+)
+
+type AppengineInfoK8s struct {
```

Comment:
If we are implementing an interface; we will want to make sure liskov substitution principle holds true; or else things are bound to break given time. If the interface does not make sense for all implementations (which this at the surface doesn't seem to be the case); it probably makes sense to start looking at a different abstraction for implementations to share.